eng/pipelines/build-all-lib.yml

```diff
@@ -5,10 +5,12 @@ trigger: none    # Disable dev and main branches.
 pr: none         # Disable pull request triggers.
 
 parameters:
-  - name: Projects  	    # List of projects to build #**/Microsoft.FluentUI.AspNetCore.Components.Emoji.csproj
+  - name: Projects  	    # List of projects to build 
     default: |
       **/Microsoft.FluentUI.AspNetCore.Components.csproj
       **/Microsoft.FluentUI.AspNetCore.Components.Icons.csproj
```

Comment:
I think you need to remove the previous `Icons.csproj` line
And maybe to add the new `ODataAdapter` project?